logic.py

```diff
@@ -858,7 +861,7 @@ def email_new_user_pending(email, name):
         ]
     }
 
-    _SENDGRID.client.mail.send.post(request_body=msg)
+    # _SENDGRID.client.mail.send.post(request_body=msg)
```

Comment:
Would be nice to wrap this behind an option or env var or something instead of commenting out entirely. But we'll take it for now...